pyproject.toml

```diff
@@ -5,11 +5,73 @@ requires = [
     "setuptools>=42",
```

Comment:
For the record, this requires feature setuptools >= 61.

https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html

Reply:
Thanks, fix is in PR #210.